e2e/test/iothub/ConnectionStatusChangeHandlerTests.cs

```diff
@@ -2,28 +2,21 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
-using System.Diagnostics.Tracing;
 using System.Threading.Tasks;
 using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.E2ETests.Helpers;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Microsoft.Azure.Devices.E2ETests
 {
     [TestClass]
     [TestCategory("E2E")]
     [TestCategory("IoTHub")]
-    public class ConnectionStatusChangeHandlerTests : IDisposable
+    public class ConnectionStatusChangeHandlerTests : E2EMsTestBase
     {
         private readonly string DevicePrefix = $"E2E_{nameof(ConnectionStatusChangeHandlerTests)}_Device";
         private readonly string ModulePrefix = $"E2E_{nameof(ConnectionStatusChangeHandlerTests)}";
-        private static TestLogger _log = TestLogger.GetInstance();
-
-        private readonly ConsoleEventListener _listener;
-
-        public ConnectionStatusChangeHandlerTests()
-        {
-            _listener = TestConfig.StartEventListener();
-        }
+        private static TestLogger s_log = TestLogger.GetInstance();
```

Comment:
Just remove this s_log. The base class has 'Logger'. Use that below.

Reply:
awesome, good point!